/**:
  ros__parameters:
    min_area: 200
    max_area: 10000
